web/packages/teleterm/src/ui/Search/searchResult.ts

```diff
@@ -18,9 +18,6 @@ import type { ClusterUri } from 'teleterm/ui/uri';
 import type { Cluster } from 'teleterm/services/tshd/types';
 
 import type * as resourcesServiceTypes from 'teleterm/ui/services/resources';
-import type { SearchResultResource } from 'teleterm/ui/services/resources';
-
-export { SearchResultResource };
```

Comment:
This export was not used so I just removed it. Babel was complaining that `SearchResoultResource` was undefined.